src/App/Resources/AppResources.resx

```diff
@@ -2214,4 +2214,7 @@
   <data name="SpecialCharacters" xml:space="preserve">
     <value>Special Characters (!@#$%^&amp;*)</value>
   </data>
+  <data name="GoBack" xml:space="preserve">
```

Comment:
IMO the key and the value should be the same, except on cases where it's too large, like descriptions. So in this case it'd be `TapToGoBack`. Furthermore, maybe we need "Go back" as a string in other place and a developer may use this string because the key is misleading, and also if we'd need "Go back" we couldn't use the key given that it'd be already in use by this.